src/local-development/code-pull-deploy.ts

```diff
@@ -175,25 +178,34 @@ export async function deployComponentCode({
 	const sourceContentUint8 = await vscode.workspace.fs.readFile(sourcePath);
 	const sourceContent = new TextDecoder().decode(sourceContentUint8);
 
-	// Get the code from the API
-	const axiosConfig: AxiosRequestConfig = {
-		url: getCodeApiUrl({ appComponentType, remoteComponentName, apiCodeType: codeDef.apiCodeType, origin }),
-		method: 'PUT',
-		headers: {
-			Authorization: 'Token ' + origin.apikey,
-			'Content-Type': codeDef.mimetype,
-			'imt-vsce-localmode': 'true',
-			'imt-apps-sdk-version': ExtensionVersion,
-		},
-		data: sourceContent,
-		transformRequest: (data) => data, // Do not expect the `data` to be JSON
-	};
-	await requestMakeApi(axiosConfig);
+	const localChecksum = md5(sourceContent.toString());
```

Comment:
Based on typing, the `sourceContent` is already a `string`. Therefore the `toString()` looks redundant.